core/src/main/java/org/bouncycastle/asn1/ASN1Set.java

```diff
@@ -472,8 +474,9 @@ private static boolean lessThanOrEqual(byte[] a, byte[] b)
          * primitive form accordingly. Failing to ignore the CONSTRUCTED bit could therefore lead to
          * ordering inversions.
          */
-        int a0 = a[0] & ~BERTags.CONSTRUCTED;
-        int b0 = b[0] & ~BERTags.CONSTRUCTED;
+
```

Comment:
Remove the new line introduced here

Reply:
85b3bb869de9e7728d82b944d4e2983aabe22366: removed newline